### PR TITLE
add LiveRenderer

### PR DESCRIPTION
**Pull request by dushan555**

### CODE REVIEW

File: live/live.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Using live as m3u8 and DLNA media renderer
+#
+# Macast Metadata
+# <macast.title>Live Renderer</macast.title>
+# <macast.renderer>LiveRenderer</macast.renderer>
+# <macast.platform>darwin,win32,linux</macast.platform>
+# <macast.version>0.1</macast.version>
+# <macast.host_version>0.7</macast.host_version>
+# <macast.author>NOTAG</macast.author>
```
Comment (xfangfang) [inline live/live.py:9]: The author name should be your github id, which is dushan555
Macast will read this value to correctly display your GitHub avatar

File: live/live.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# <macast.desc>Live support for Macast</macast.desc>
```
Comment (xfangfang) [inline live/live.py:10]: It would be better if the description text is more detailed.

File: live/live.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    @property
+    def renderer(self):
+        renderers = cherrypy.engine.publish('get_renderer')
+        if len(renderers) == 0:
+            logger.error("Unable to find an available renderer.")
+            return None
+        return renderers.pop()
+
+    def on_start_click(self, item):
+        print(item.text, item.data)
```
Comment (xfangfang) [inline live/live.py:71]: It is recommended to use `logger` to output content.

```python
logger.info(f'text: {item.text} data: {item.data}')
```

File: live/live.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        self.renderer.start_url(item.data)
```
Comment (xfangfang) [inline live/live.py:72]: Why not use the default playback method of Renderer?

```python
self.renderer.set_media_url(item.data)
```

**Review by xfangfang — COMMENTED** [review]
Great plugin !!!
I left some comments for reference. There may be some places that need to be changed to avoid possible problems.

**Comment by dushan555** [comment]
> Great plugin !!! I left some comments for reference. There may be some places that need to be changed to avoid possible problems.

Thank you for your reply. I will revise the submission later

**Pull request merged**